Desktop/data/columnsmodel.cpp

```diff
@@ -16,7 +16,7 @@ ColumnsModel::ColumnsModel(DataSetTableModel *tableModel) : QTransposeProxyModel
 
 QVariant ColumnsModel::data(const QModelIndex &index, int role) const
 {
-	if(index.row() < 0) return QVariant() ; // index.row() >= rowCount()): the row can be greater than the rowCount when a data of a column is asked.
+	if(index.row() < 0 && index.row() >= rowCount()) return QVariant();
```

Comment:
How could that ever be true?

Reply:
net aangepast voordat ik jouw comment zag...